src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -1046,6 +1046,16 @@ else if (block instanceof BlockFlowerPot)
             }
         }
 
+        if(block instanceof BlockChest && job instanceof JobBuilder)
+        {
+            BlockPos buildingLocation = ((JobBuilder) job).getWorkOrder().getBuildingLocation();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'buildingLocation' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)